### ISSUE TITLE
Format Techtonica Slideshows Part 2/3

### ISSUE THREAD

**Issue by alodahl** [Techtonica/curriculum#599]
This single-contributor task is to ensure all Techtonica-owned slideshows have the same styles and formatting. This a simple task, but may take a while! Some of these are already formatted, but your job is to double-check or make changes.

uncheck all boxes before you start. Let Alina know if you can’t.
Check the box next to the slideshow once it is ready.
Message me (Alina) the numbers of any slideshows that are locked to editing, and I will unlock them for you.
Here is an example of a slideshow with correct formatting. https://docs.google.com/presentation/d/1ANf64yQ_Nxtul45xofh8cpjWF23UM6c8m8kJQHQyx_Q/edit?usp=sharing
Notice:

- Each slide has the Techtonica logo & bar at the bottom
- The accepted text colors are black, gray, and custom turquoise
- All text is in Lato font
- Headers are at size 28 font, while regular text is either 18 or 24(preferred), with the exception of footnotes/side notes
- sections are continued on additional slides if things get too cramped
- Capitalization, punctuation, and alignment are consistant
- This list is not exhaustive - we need someone detail oriented 🕵️‍♀️

--------------------------------

- [ ] 31. https://drive.google.com/open?id=1MWgf17vIk47MfOjIdCfrE4qAbdDsU7OH94H0owQKT8Q
- [ ] 32. https://docs.google.com/presentation/d/1_QyXF5TGja-PhSgJUiuFKqLafbyQ_q3gvzEbDMkdBIg/edit?usp=sharing
- [ ] 33. https://docs.google.com/presentation/d/1b0WNXL2pcw7XwcuF2k4M0ZqzIt-iUpnAQiQZ60BzyFI/edit?usp=sharing
- [ ] 34. https://docs.google.com/presentation/d/13ROF6Eq4sQwUMFOwDYQPxvCw-vhz5A-Q5gACalNXQyU/edit?usp=sharing
- [ ] 35. https://drive.google.com/open?id=1dGsFDog3uUq0XwVMCbYRJucPuzBWTFCawdas2r6fBdA
- [ ] 36. https://drive.google.com/open?id=1RSi5VLfmObQTtPerd-fDl4s3Sof3oT09cKKr3bfjl2s
- [ ] 37. https://drive.google.com/open?id=18CJBM1Z6IA4MGGy6phf2bJN5gOrzQ64y1coIMcq8te0
- [ ] 38. https://drive.google.com/open?id=1USOZJSzwXmSYepjwrE9r9ky_fmQ8VPE7bshjsBoDsZM
- [ ] 39. https://www.netlify.com/blog/2016/10/27/a-step-by-step-guide-deploying-a-static-site-or-single-page-app/
- [ ] 40. https://drive.google.com/open?id=1WIm5UCQL9TOsmW5X6suapBhyEqtk8Y2oLKb5gUqbzus
- [ ] 41. https://drive.google.com/open?id=10FUe4xQHrWOc7Ys3bWUFJDvM4IhYxMX5AgmPY8cxjlM
- [ ] 42. https://docs.google.com/presentation/d/1Bs6kGlzVK6chplOHFH9OiCBQj1kQTzrmCL0cHcaZD8U/edit#slide=id.p
- [ ] 43. https://docs.google.com/presentation/d/1Pj-hkQCeVe4kJJ4s8RHcSZrNKx4Mp4kaQ9J0Eyy1gAQ/edit#slide=id.p
- [ ] 44. https://drive.google.com/open?id=1yjSpOpwPbVtl5K8QbQvtK5t6bn5wtmZDpBgX9v_vkD4
- [ ] 45. https://drive.google.com/open?id=1-TEyN4TTjP9c9WGcThS1jQmVoY4pG-xt9gjbDhJkUTc
- [ ] 46. https://drive.google.com/open?id=1hmP5pOfrgexspDS827_cGkb3SNqhIJnbxrlnRFcHCGw
- [ ] 47. https://drive.google.com/open?id=1N2eDw84BqmcqvNDjtQfNEF_7PO91z-IHTR44QXt3-oI
- [ ] 48. https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-3.md
- [ ] 49. https://docs.google.com/presentation/d/1ZcOdekB_aP59huZdp4X0u6EfUJKgxzK7y8LqCmzSLC8/edit?usp=sharing
- [ ] 50. https://docs.google.com/presentation/d/1XKxBZIJ2gLknsyCBFm0jcq5Gk06d9uAyYTSW0QmQA10/edit?usp=sharing
- [ ] 51. https://docs.google.com/presentation/d/1rk4IfcH86swcqFeBV7eHJAvLEN2nbzkjwrnAjYWBt-s/edit?usp=sharing
- [ ] 52. https://drive.google.com/open?id=1viDJyLHXVs-VioUEatvWHDZeOCCmPxZNPHzH0C393rg
- [ ] 53. https://docs.google.com/presentation/d/1jMkVbUkZS40z7kAoC1mPIt8RwfF8-GRL8oeee25UzYw/edit?usp=sharing
- [ ] 54. https://docs.google.com/presentation/d/16X4u-tyy_Vdo7lp3jUEXAsi24lpkQ6H5GYVxqWI0s3c/edit#slide=id.g1f8128643f_0_22
- [ ] 55. https://drive.google.com/open?id=1Ol1as_RuxBpXMd4VxCUyvFVkjpcqAxg6B-c0Sl9KDLc
- [ ] 56. https://drive.google.com/open?id=1sD3nwQnhbe1wPnAWes0Nbt578tJacTtx0Yqy8XFp7w8
- [ ] 57. https://drive.google.com/open?id=1S3BjcLZNjex2_qiA9MdyJOjWZ_qmJ78STbUeDEyHH_8
- [ ] 58. https://docs.google.com/presentation/d/1Vzx0eyKt0jNQq70pnAeDQ5X1qOPdjIzCgujSp6dd_1U/edit?usp=sharing
- [ ] 59. https://drive.google.com/open?id=1fMwUt4X5W5rps0jmThOT6iP1Ky3R8EBniPA4TY74jxQ
- [ ] 60. https://drive.google.com/open?id=12h5SyQO7yk-cepXk_t5o-5zvXLGAOUFU-Ui7U5ZLDcY
- [ ] 61. https://docs.google.com/presentation/d/1SZAzZBRiwmQyVnxGR8d1pfEaG265Z9mU2EFqUizqHUQ/edit#slide=id.p

**Comment by Jayashree-panda** [comment]
@alodahl #32. The last 2 slides don't have the bar and Techtonica logo at bottom. Can I solve this issue for gssoc 2019?

**Comment by Kundan28** [comment]
Yes, you can solve that issue for GSSoC19. But recheck if there anything else missing in other slides too.

**Comment by alodahl** [comment]
@Jayashree-panda please be aware that for this issue, you must take responsibility for the entire list of slide links to get the "HARD" level points for your PR.

**Comment by Jayashree-panda** [comment]
@alodahl yes sure. I will check all the slides and edit it.

**Comment by pydevsg** [comment]
@Jayashree-panda are you working on this?
Else I would like to take this issue ?

**Comment by alodahl** [comment]
@pydevsg: @Jayashree-panda already took this issue, but I was able to set aside #593 for you.

**Comment by pydevsg** [comment]
Okay, thanks for the response,Mam.

**Comment by Jayashree-panda** [comment]
@alodahl I don't have any access to edit these presentations.

**Comment by alodahl** [comment]
just tell me which presentations you need to edit but don’t have access to, and I will add you. Send me your gmail address in slack and remind me!

**Comment by alodahl** [comment]
Hi @Jayashree-panda How is this issue going?

**Comment by Jayashree-panda** [comment]
Hey! Sorry for the delay.
These are the slides I would like to edit. Please unlock it.
32,36,37,45,49,50,54,61

**Comment by alodahl** [comment]
Hi @Jayashree-panda, slideshows 32 & 49 are locked for me as well, so you'll have to click "view only" > "request edit access" and say Alina told you to edit these in the message.  My boss should grant you access.  The others you asked about should be editable for you now.

**Comment by Jayashree-panda** [comment]
@alodahl Yes, I have requested your boss to provide me the access. Meanwhile I am editing the rest.

**Comment by Jayashree-panda** [comment]
@alodahl I also don't have access to edit slide 50,51,64

**Comment by alodahl** [comment]
@Jayashree-panda You should get an invitation to 50, 51, and 54.  64 is not part of this issue 😉

**Comment by Jayashree-panda** [comment]
@alodahl I have edited the slides. Please have a look.

**Comment by alodahl** [comment]
@Kundan28  @manaswinidas @hoduchieu01 @saberprashant, if someone is making a PR for something not in the repo, like the formatting slides assignments or a research task -
- Please review their work on the issue - clearly the single typo that is the PR does not matter.  Tell them if they have not fullfilled what is requested in the issue. If they have, you may approve it and mark the PR # in the scoresheet.
- If it is something hard to review, like 30 slideshows, please take 5 at random and make sure that the formatting guidelines are being followed. If the 5 you chose are correct, you can approve the issue.  If several of us do this, it should be a pretty good assessment.

**Comment by alodahl** [comment]
@Jayashree-panda, have you finished this issue?  If so, please make a PR by changing 1 typo in a single file, make a pull request with that, but make the title and description about issue 599.

**Comment by alodahl** [comment]
If we don't hear from you in a few days, we will put this issue back in to-do.

**Comment by imskr** [comment]
Can I take this?

**Comment by imskr** [comment]
I need access to slide no: 31, 32, 33, 34, 38, 41, 42, 43, 44, 46, 47, 49, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60 @alodahl

**Comment by Kundan28** [comment]
@imskr I have reviewed slideshow  no. 35, 36, 37, 40, 45 & 50 and commented all the changes to be made.

**Comment by imskr** [comment]
Thanks @Kundan28!  I haven't started with this issue yet!  Just waiting to get access to all then I will start.

**Comment by Kundan28** [comment]
> Thanks @Kundan28! I haven't started with this issue yet! Just waiting to get access to all then I will start.

You can work on rest of them whose access you have right now...as it will save a lot of time.

**Comment by alodahl** [comment]
@imskr how is this issue going? Do you need anything?

**Comment by imskr** [comment]
Sorry for the delay @alodahl I will finish this today. 😁
